packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -0,0 +1,446 @@
+
+##### Keyboard navigation
+
+The Form Pill Group is focusable, but only one pill is focusable at a time. This means the Pill Group is a single tab stop to a keyboard user. Once focused within the Pill Group the following keyboard interactions apply.
+
+- Left and Right arrow keys move focus within the group
+- If a pill is selectable, spacebar and enter will toggle pill selection
+- If a pill has a supplied action via `onClick`, enter key will trigger that action
+- If a pill is dismissable, the pill can be removed by pressing the delete or backspace key
+
+<Box marginBottom="space70">
+  <AspectRatio ratio="16:9">
+    <iframe
+      src="https://www.loom.com/embed/1bea986ec6a24cd494950e2c12f10562"
+      frameborder="0"
+      webkitallowfullscreen
+      mozallowfullscreen
+      allowfullscreen
+      title="Silent demo on keyboard navigation"
+    ></iframe>
+  </AspectRatio>
+</Box>
+
+#### Display Pill vs Form Pill
+
+There are some very important semantic differences between a Display Pill and Form Pill group. Below is a table explaining the main differences to get a better understanding of when to use each type of Pill.
+
+<FormPillVsDisplayPillTable />
+
+## Examples
+
+### Basic Form Pill Group
+
+The basic Form Pill Group can be composed by adding `FormPill` components as children of `FormPillGroup`. These must be used with the `useFormPillState` hook, and the returned state should be spread onto each component.
+
+This will provide you with some internal state management and keyboard navigation.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {defaultExample}
+</LivePreview>
+
+### Selectable Form Pill Group
+
+If you would like to make the Pill Group have selectable pill, you should manage the selection state yourself and combine it with the state returned from the `useFormPillState` hook.
+
+To do so, track which pill is selected in a separate store of state. When rendering the Pill Group, cross reference the rendered pills with the selection state, and conditoinally set `selected` on each `FormPill` that requires it.
+
+The `onSelect` event handler will fire when ever the pill is clicked, or the spacebar or enter key is pressed. Use this to respond to your users selection interactions.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {selectableExample}
+</LivePreview>
+
+### Removable Form Pill Group
+
+Form Pills are given a closing x button when provided an `onDismiss` event handler. Because the Form Pill Group is largely presentational, you can provide "removable" functionality by managing the state of the rendered Pills. By responding to user interactions and hooking into the `onDismiss` event handler, the rendered state of the Form Pill Group can be updated and pills can be selectively removed from the collection.
```

Comment:
```suggestion
Form Pills are given a closing "x" button when provided an `onDismiss` event handler. Because the Form Pill Group is largely presentational, you can provide "removable" functionality by managing the state of the rendered pills. By responding to user interactions and hooking into the `onDismiss` event handler, the rendered state of the Form Pill Group can be updated and pills can be selectively removed from the collection.
```